recipe/meta.yaml

```diff
@@ -22,15 +22,18 @@ requirements:
   host:
     - python >=3.7
     - pip
+    - setuptools_scm
```

Comment:
This isn't listed as a build dependency but this line suggests that it's probably needed:
https://github.com/astanin/python-tabulate/blob/v0.9.0/pyproject.toml#L37

Reply:
Yeah, I overlooked it in the build system section:
https://github.com/astanin/python-tabulate/blob/master/pyproject.toml#L2

Reply:
grayskull seems to have missed it, too.  That's disappointing.

Reply:
https://github.com/conda-incubator/grayskull/issues/340

Reply:
Looks like `pyproject.toml` just isn't fully supported by grayskull yet.